# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mythtv/fribidi-include.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mythtv-0.21/libs/libmythtv/libmythtv.pro.orig	2008-02-18 23:59:32.000000000 +0100
+++ mythtv-0.21/libs/libmythtv/libmythtv.pro	2008-09-09 22:14:42.000000000 +0200
@@ -240,7 +240,10 @@
     using_ivtv:SOURCES += ivtvdecoder.cpp
 
     # On screen display (video output overlay)
-    using_fribidi:DEFINES += USING_FRIBIDI
+    using_fribidi: {
+        DEFINES += USING_FRIBIDI
+        QMAKE_CXXFLAGS += $${FRIBIDI_CFLAGS}
+    }
     HEADERS += osd.h                    osdtypes.h
     HEADERS += osdsurface.h             osdlistbtntype.h
     HEADERS += osdimagecache.h          osdtypeteletext.h
